db/schema.rb

```diff
@@ -347,125 +347,125 @@
     t.bigint "responder_id"
     t.bigint "creator_id"
     t.bigint "jurisdiction_id"
-    t.binary "submission_token", limit: 255
+    t.binary "submission_token", limit: 200
```

Comment:
I think your current local db state might need to be updated with `1.41.0` before this migration, we removed a migration that changed these limits to `200` awhile ago

Reply:
Thank you! See [88cc595949ec2126aac05e16bfda99ab100b1a0c](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/88cc595949ec2126aac05e16bfda99ab100b1a0c)